### ISSUE TITLE
使用quicker进行按键映射

### ISSUE THREAD

**Issue by lbgws2** [soluty/sui#4]
我之前使用过autohotkey作为按键映射方案,但是AHK的晦涩的语法使用起来非常不舒服

后来我使用quicker作为按键映射,按键映射非常直观,也支持鼠标手势,也可以自定义脚本,可否加入对quicker的支持

**Comment by soluty** [comment]
是的, ahk的语法垃圾的不行, 只是windows上我没找到可以替代它的, python或许可以? 我了解了一下quicker, 它好像没有自己的脚本语法, 只是可以执行bat或者ahk的脚本. 不知道我理解的对不对, 如果是这样, quicker没太大的优势, 这个项目主要是提出一个快捷键设计的理念, 至于实现用什么则无所谓, 如果我用quicker再重新实现一遍, 我看不到它的意义何在, 你如果认可sui的快捷键设计, 也可以自己用quicker来实现, 有一个感兴趣的朋友, 正在macos通过hammerspoon来实现sui呢.
